api/groups/addressGroup.go

```diff
@@ -20,6 +20,7 @@ const (
 	getAccountPath            = "/:address"
```

Comment:
I do not think we need this new route because on /address/:address route we return the api.AccountResponse struct that contains the codehash already.

Reply:
Yes but it will return all the account data, including the hex code, so a lot of data is going to be returned whereas I just want the code hash.

As an example, on a page, I may need to request 10 code hashes of 10 different smart contracts. Currently, the data sent over the wire is 100x superior to what could be achieved if only the code hash was sent.

Furthermore, if we following the reasoning "we don't need this route because /address/:address returns everything", why having added "/address/:address/balance" or "/address/:address/username"?

Reply:
agree, it is ok to add this endpoint. But keep in mind that `development` branch is used for future releases, therefore your addition won't go live too soon. However, it is the only way we can accept your PR, because the release candidate branch is in final testing procedures.
Also, please use signed commits. Otherwise, we won't be able to merge your changes

Reply:
ok, will review the code then